app/services/invite_schools.rb

```diff
@@ -261,16 +262,16 @@ def invite_sitless_not_opted_out_schools_for_nqt_plus_one
       .not_opted_out
       .where.missing(:induction_coordinators)
       .find_each do |school|
-        if school.contact_email.blank?
```

Comment:
No idea what's going on here

Reply:
I think `School` is indented with 4 spaces, each of the scopes is indented with 6, and body of the block is indented 2 deeper than the start of previous expression, rather than start of previous line. For some reason.

I guess if we made the school scope into a single line it would look reasonable...